src/agenda/platform-style.ts

```diff
@@ -1,4 +1,9 @@
-export default function platformStyles(appStyle) {
+interface AppStyle {
+  agendaKnobColor: string;
+  calendarBackground: string;
+}
+
+export default function platformStyles(appStyle: AppStyle) {
```

Comment:
Same here. Try using the new Theme type instead of the declaring a new AppStyle type